cognite/client/data_classes/data_modeling/instances.py

```diff
@@ -66,15 +66,15 @@ def load(cls, data: dict) -> NodeOrEdgeData:
         return cls(**convert_all_keys_to_snake_case(data))
 
     def dump(self, camel_case: bool = False) -> dict:
-        output = asdict(self)
+        output = {"properties": dict(self.properties.items())}
         if self.source:
             if isinstance(self.source, (ContainerId, ViewId)):
-                output["source"] = self.source.dump(camel_case)
+                output["source"] = self.source.dump(camel_case)  # type: ignore[assignment]
```

Comment:
Instead of type ignore, you can declare the type of `output` as `dict[str, Any]`